### DISCUSSION TITLE
GitHub pro subscription ended while still a student

### DISCUSSION THREAD

**Discussion by literallyfiro** [community/community#42063] [GitHub Education] [answered]
I just got an email saying that my GitHub Pro coupon has ended and I should update my payment method, but I have the github students pack. What should I do to regain my GitHub pro license?

I also checked the GitHub education application and it said I'm still a student

**Comment by DSDmark** [comment, marked as answer]
Hi @ImOnlyFire,

you received an email saying that your GitHub Pro coupon has ended, it is possible that your Student Developer Pack membership may have expired or been terminated.

To regain access to the benefits of the Student Developer Pack, including the GitHub Pro license, you will need to reapply for the Student Developer Pack. You can do this by visiting the GitHub Education website (https://education.github.com/) and following the steps to apply for the Student Developer Pack. If you are eligible for the Student Developer Pack After reapply it. You should receive confirmation of your membership within a few days.

**Reply by literallyfiro** [reply]
Hello, I have already tried to reapply for the Students developer pack, but when I select my email it says "[email] has already been used to obtain an academic discount on GitHub account(s) ImOnlyFire". 

Should I use another email address to reapply?

**Reply by DSDmark** [reply]
If you are seeing a message that your email address has already been used to obtain an academic discount on your account, it may be because you have already applied for the Student Developer Pack and received the discount. In this case, you should not need to reapply.

**Reply by literallyfiro** [reply]
But then how should I be able to regain my github pro license?

**Reply by literallyfiro** [reply]
Okay, it was actually a bug on the phone version of the website. I applied from my computer and it worked. Thank you

**Reply by zxcoder47** [reply]
@literallyfiro Hello I also facing this problem can plz guide me on how you can solve this problem in my case they say "We're still processing your application. Once this is complete you will have access to the student developer pack offers."

**Reply by samus-aran** [reply]
Hi @zxcoder47  👋

We’re happy you’re here but let’s not divert this discussion as this conversation is off-topic. Please open up a separate discussion so that we can look at your query there. 👍
